lodash.js

```diff
@@ -9000,14 +9000,12 @@
      * jQuery(window).on('popstate', debounced.cancel);
```

Comment:
Sure.  Let me add an additional commit that adds some more detailed comments.  I could also walk you through the code in a screenshare.